pkg/controllers/cloud/networkpolicy_sync.go

```diff
@@ -291,6 +291,7 @@ func (r *NetworkPolicyReconciler) syncWithCloud() {
 			}
 		}
 	}
+	log.V(1).Info("Synchronization content is nil")
```

Comment:
@Anandkumar26 as discussed offline, could you please add 2 info logs to figure out start and end of cloud sync. This is to understand from logs the course of action taken place in cloud sync. 
Also, in line 278, could you please explicitly state that SG is not present in cache, hence deleting it?